charts/management-portal/templates/deployment.yaml

```diff
@@ -47,14 +47,20 @@ spec:
           - name: SPRING_PROFILES_ACTIVE
             value: "prod,swagger"
           - name: SPRING_DATASOURCE_URL
-            value: "jdbc:postgresql://{{ .Values.postgres.host }}:5432/managementportal"
+            value: "jdbc:postgresql://{{ .Values.postgres.host }}:5432/managementportal?ssl=true&sslmode=verify-ca"
```

Comment:
SSL configuration is not true in general. Perhaps instead of a postgres host, this should take a JDBC URL, e.g. `{{ .Values.postgres.url }}`.

Reply:
Fixed in fbda1ed